mlflow/java/client/src/main/java/org/mlflow/tracking/MlflowClientVersion.java

```diff
@@ -0,0 +1,34 @@
+package org.mlflow.tracking;
+
+import java.io.InputStream;
+import java.util.Properties;
+
+import com.google.common.base.Supplier;
+import com.google.common.base.Suppliers;
+
+/** Returns the version of the MLflow project this client was compiled against. */
+public class MlflowClientVersion {
```

Comment:
Nit: could this be a protected class since it's internal / only used by other classes in org.mlflow.tracking?